imod/mf6/auxiliary_variables.py

```diff
@@ -25,11 +25,40 @@ def _get_auxiliary_data_variable_names_mapping(package):
     return result
 
 
-def add_periodic_auxiliary_variable(package):
+def add_periodic_auxiliary_variable(package) -> None:
+    """
+    splits an auxiliary dataarray (with one or more auxiliary variable dimension) into dataarrays per
+    auxiliary variable dimension. For example a concentration auxiliary variable in a flow package
+    will have a species dimension, and will be split in several dataarrays- one for each species.
+    """
     if hasattr(package, "_auxiliary_data"):
```

Comment:
with the interface and property in place as described above you don't need this if statement anymore and can directly iterate over 

`for aux_var_name, aux_var_dimensions in package.auxiliary_data.items():`

This also applied to the method below